src/angular/components/send/add-edit.component.ts

```diff
@@ -237,6 +237,7 @@ export class AddEditComponent implements OnInit {
             this.platformUtilsService.showToast('success', null, this.i18nService.t('deletedSend'));
             await this.load();
             this.onDeletedSend.emit(this.send);
+            return true;
         } catch { }
```

Comment:
You need at least one fallback `return false;` at the end of the method since you have a `try { ... } catch { }`

Reply:
That feeling when you don't save your final changes before pushing them upstream....



Thank you :D